### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by sashaperigo**
Hey Tony,

Sorry for the obnoxious, large, and disorganized pull request here! I've done by best to outline all major changes below.

**Gallery Page**
new gallery page
I went ahead and added some context at the top of the page as to what these images were! I also added some posters and deleted a few that we're less proud of. Eventually I want to add code to gray each image out on mouseover and display some more info about the show, but I obviously haven't implemented that yet.

**Styling**
I made a couple quick changes to the styling which I like a little better! Let me know how you feel. I added some additional space between the logo and the "Meet The Team" entry on the nav bar, updated the default font on Apple devices, and made the little line that appears under each nav bar selection thinner.

**Team Data**
I alphabetically ordered the names within each category on the team page and updated some people's information.

Thanks!
Sasha

### CODE REVIEW

File: gallery.html [hunk 1]
```diff
@@ -3,17 +3,13 @@
...
+    </div>
+    {% for image in site.static_files %} {% if image.path contains 'img/posters' %}
+    <a href="{{ image.path }}">
+        <img src="{{ image.path }}">
+    </a>
+    {% endif %} {% endfor %}
 </div>
-
-<link href="//cdnjs.cloudflare.com/ajax/libs/lightgallery/1.2.21/css/lightgallery.min.css" rel="stylesheet">
-<script src="//cdnjs.cloudflare.com/ajax/libs/lightgallery/1.2.21/js/lightgallery.min.js"></script>
```
Comment (tonybruess) [inline gallery.html:16]: Why did you remove these?

**Comment by tonybruess** [comment]
Thanks for the lovely PR!

File: _data/team.yml [hunk 2]
```diff
@@ -1,22 +1,22 @@
...
-    artist: Band of Horses
-    artist_link: https://open.spotify.com/artist/0OdUWJ0sBjDrqHygGUXeCF
+Directors:
   - image: tommy
     name: Tommy Choi
-    position: Co-President
+    position: Co-Director
     year: Junior
-    email: tjchoi@stanford.edu
+    email: tjchoi
```
Comment (tonybruess) [inline _data/team.yml]: Why remove the @stanford.edu?

Comment (sashaperigo) [reply _data/team.yml]: Sorry, forgot to add an explanation for this! Some of the emails were getting cut off on my browser, and I couldn't figure out any way to fix this without making the font super tiny. I tried out this solution -- removing the @stanford.edu but leaving the click-through links the same -- because I figured it served the same purpose.

This is admittedly a sub-optimal solution. If you have a better idea lmk!

File: gallery.html (hunk 1 above)
Comment (sashaperigo) [reply gallery.html:16]: Oops, this was an accident! Fixing now.

**Comment by tonybruess** [comment]
I put the email addresses back in and added a `truncate` filter to the team page so they don't overflow. Without the full address, the `mailto:address` is missing the `@stanford.edu` and doesn't work.

**Comment by sashaperigo** [comment]
@tonybruess thanks for that! 😊
